resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

```diff
@@ -0,0 +1,82 @@
+
+Hello les L2 ||<@&794628669737009183>|| !
+
+Avec la publication des classements des 3 premiers semestres et le choix de la mobilité qui arrive à grand pas,
+[Un formulaire circule, permettant à tout le monde de se pré-situer (ça reste anonyme, ne vous inquiétez pas)](https://forms.gle/5oC6osJFc3EXDz1H6) (créé par <@689931471430877233>)
+Les résultats sont disponibles après avoir l'avoir rempli.
+Partagez le au max comme ça on peut tous avoir une idée d'où on peut se retrouver l'an prochain 🙂
+
+Merci!
+
+
+**__INFOS POUR LES L3: SEMESTRE INTERNATIONAL S5__**
+
+Ces informations vous permettront de les comparer plus facilement :map:
+N'hésitez pas à poser vos **questions relatives aux destinations sur <#889180185574924328>** (ceux qui ont déjà voyagé dans vos futures destinations pourront vous y répondre) !
+**:warning: TOUTES LES DONNÉES DE COÛTS ET DE CLASSEMENT DATENT DE SEPTEMBRE 2021 :warning:
+POUR TOUTES LES DONNÉES MANQUANTES, à CE JOUR NOUS NE SAVONS PAS SI NOUS LES AVONS**
+
+**Choix final le  [LIEN A VENIR]**
```

Comment:
```suggestion
**Choix final le  [LIEN A VENIR]**
```
il manque le lien

Reply:
Ik je sais pas 
et j'ai pas d'info 

mais on peut l'enlever (for now)
A moins qu'ils vous ont donné le sondage moodle où il faudra faire le choix

Reply:
non on a rien pour le moment, autant l'enlever alors je pense 😄